File: app/src/main/java/com/juanmacuevas/shoottheflakup/Landscape.java
package com.juanmacuevas.shoottheflakup;

import android.content.res.Resources;
import android.graphics.Bitmap;
import android.graphics.BitmapFactory;
import android.graphics.Canvas;
import android.util.DisplayMetrics;

class Landscape {
    private Bitmap mBackgroundImage;
    public Landscape(Resources res, DisplayMetrics metrics) {
        mBackgroundImage = BitmapFactory.decodeResource(res, R.drawable.background);
        mBackgroundImage=Bitmap.createScaledBitmap(
                mBackgroundImage, metrics.widthPixels, metrics.heightPixels, true);
    }

    public void draw(Canvas canvas) {
        canvas.drawBitmap(mBackgroundImage, 0, 0, null);
    }
}

File: app/src/main/java/com/juanmacuevas/shoottheflakup/Landscape.kt
package com.juanmacuevas.shoottheflakup

import android.content.res.Resources
import android.graphics.Bitmap
import android.graphics.BitmapFactory
import android.graphics.Canvas
import android.util.DisplayMetrics

internal class Landscape(res: Resources, metrics: DisplayMetrics) {
    private var mBackgroundImage: Bitmap? = null

    init {
        mBackgroundImage = BitmapFactory.decodeResource(res, R.drawable.background)
        mBackgroundImage = Bitmap.createScaledBitmap(
            mBackgroundImage!!, metrics.widthPixels, metrics.heightPixels, true
        )
    }

    fun draw(canvas: Canvas) {
        canvas.drawBitmap(mBackgroundImage!!, 0f, 0f, null)
    }
}
